### DISCUSSION TITLE
Add scope to sort products by lowest price/highest price.

### DISCUSSION THREAD

**Discussion by jackabox** [statamic-rad-pack/shopify#7] [Feature Requests]
Useful for if we are wanting to sort the front-end. Let's see if we can integrate a way to sort the product's by lowest/highest price easily. This should allow more customisation over the overall front-end and be better for the end-user who will be using the website.

**Comment by austriker27** [comment]
Yes I like this idea a lot. 

I think sorting in general is an area for improvement, I haven't deep dived on it yet... As an example my current site has the following sorting ability: 
![image](https://user-images.githubusercontent.com/14855925/117978579-e2123b80-b2e6-11eb-9b23-dcef0f35b1d8.png)


It would be nice if we had some of that, most relevant would: price, popularity (sales within last 30? days), rating, total sales. And extra nice if we could add these filters on the frontend to allow customers to sort and filter

**Reply by jackabox** [reply]
There isn't a way we track sales within this, that's down to Shopify's side, so not sure how we'd tackle doing the sort by popularity/rating.

Latest should be easy within Statamic as well and shouldn't need intervention from my side.

**Reply by austriker27** [reply]
Shoot, does the shopify API send the sale data at all? Maybe instead of popular it could be a sort by most sales ever 🤔

**Comment by ryanmitchell** [comment]
In Statamic 4 you would make a Computed Value to get the base price of the product (from its variants), and then sort this by a query scope. We should leverage core Statamic functionality as much as possible.
